docs/concepts/notifications.md

```diff
@@ -92,7 +92,29 @@ function.run(handler=handler, notifications=[notification])
 ```
 
 ## Configuring Notifications For Pipelines
-For pipelines, you configure the notifications on the project notifiers. For example:
+To set notifications on pipelines, supply the notifications in the `run` method of either the project or pipeline.
+For example:
+```python
+notification = mlrun.model.Notification(
+    kind="webhook",
+    when=["completed","error"],
+    name="notification-1",
+    message="completed",
+    severity="info",
+    secret_params={"url": "<webhook url>"},
+    params={"method": "GET", "verify_ssl": True},
+)
+project.run(..., notifications=[notification])
+```
+
+### Remote Pipeline Notifications
+In remote pipelines, the pipeline end notifications are sent from the MLRun API. This means you don't need to watch the pipeline in order for its notifications to be sent.
+The pipeline start notification is still sent from the SDK when triggering the pipeline.
+
+### Local and KFP Engine Pipeline Notifications
+In these engines, the notifications are sent locally from the SDK. This means you need to watch the pipeline in order for its notifications to be sent.
+This is a fallback to the old notification behaviour and so not all the new notification features are supported. Only the notification kind and params are taken into account.
```

Comment:
This is a fallback to the old notification behavior, therefore not all of the new notification features are supported. Only the notification kind and params are taken into account.